### DISCUSSION TITLE
Support .lottie file extension

### DISCUSSION THREAD

**Discussion by hosamhos60** [elementor/elementor#22995] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

Lottie widget import .json file format only which is huge in size.

### Describe the solution you'd like

.lottie are 10x smaller in size compared to json file format. 

Open https://lottiefiles.com/ and download the same file with json and .lottie file format you will notice the difference

**Comment by PharosDigital** [comment]
.Lottie is indeed a must have now to decrease the page size.
There is a plugin that works with Elementor, and supports dotlottie : https://wordpress.org/plugins/am-lottieplayer/
But it would be really cool to support this feature natively indeed.
